### ISSUE TITLE
Suggestion - Hoverboards

### ISSUE THREAD

**Issue by HollaYalla** [InZidiuZ/op-framework-issue-tracker#3618]
### Assurance

I have read the `README.md`

### Summary

So skateboards are fun and all but we should get hoveboards they will cost 2-3x maybe let them decay slowly over time as their "battery" dies and if u have a battery in your inventory you can recharge it

a good model ive found is this

https://sketchfab.com/3d-models/black-hoverboard-dbdba4424cc341b88f8857b4afa1c5ee

it would be a nice addition and it would be a little faster than a skateboard but the battery system can cause it to be more balanced aswell as a better foot mobility

### Reason

Funs

**Comment by coalaura** [comment]
No, due to the way skateboards work this is not really doable. Also way too much work for an item thats a copy of another item
